controller/cluster_check.go

```diff
@@ -30,6 +30,13 @@ func ObjectInCluster(cluster string, obj interface{}) bool {
 			}
 		}
 	}
+	if clusterName == "" {
+		if c := getValue(obj, "Annotations").MapIndex(reflect.ValueOf("field.cattle.io/projectId")); c.IsValid() {
```

Comment:
If the object does not have `Annotations` field this may panic.  Just happens that all objects have it, but I'd rather check

Reply:
thank you @ibuildthecloud, updated